### PR TITLE
PSMDB-943 Release notes 5.0.3

### PR DESCRIPTION
**Pull request by nastena1606**
Added warning to 5.0.2

modified:   source/conf.py
	modified:   source/release_notes/5.0.2-1.rst
	new file:   source/release_notes/5.0.3-2.rst
	modified:   source/release_notes/index.rst

### CODE REVIEW

File: source/release_notes/5.0.3-2.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+   Due to many core changes to WiredTiger and the core server introduced to  facilitate new features such as resharding, time series collections, and API versioning, MongoDB 5.0.x is still unstable. We do not recommend using this release candidate for production environments. Unless you are already running a version of MongoDB 5.0.x in your production environments, consider waiting a while longer for any other issues to be worked through.
+
+
+.. warning::
+
+   Beginning with MongoDB 5.0.1, several data impacting or corrupting bugs were introduced. Details are listed below.
+
+   These bugs are fixed in MongoDB 5.0.3. Percona Server for MongoDB 5.0.3-2 includes the upstream fixes of these bugs.
+
+   Please upgrade to MongoDB 5.0.3 or Percona Server for MongoDB 5.0.3-2 as soon as possible.
```
Comment (spron-in) [inline source/release_notes/5.0.3-2.rst]: When I was quickly going through the notes this caught my eye. I believe the message we are trying to send is upgrade to 5.0.3-2 if you already have some other 5.0.X, right? 

Isn't it the same from above "Unless you are already running a version of MongoDB 5.0.x in your production environments, consider waiting a while longer for any other issues to be worked through."?

Let's keep one of them.

File: source/release_notes/5.0.3-2.rst [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+   .. admonition:: Additional upgrade planning note
+
+      If you are running an early 4.4.x version of MongoDB, remain on that version for now – specifically version 4.4.1. Version 4.4.1 – released Sep 9, 2020 was the last version prior to building in or backporting many of the features that have created the bugs mentioned above.
+
+      * If you are running any version of MongoDB or Percona Server for MongoDB above v4.4.1, please upgrade to version 4.4.9 immediately. MongoDB version 4.4.9 and `Percona Server for MongoDB 4.4.9-10 <https://www.percona.com/doc/percona-server-for-mongodb/LATEST/release_notes/4.4.9-10.html>`_ includes back-ported fixes for many of the bugs introduced with work towards v5.0.x or backports to v4.4.x for that work.
```
Comment (spron-in) [inline source/release_notes/5.0.3-2.rst]: I would merge these two into one abstract. They are contradictory - we say stay on 4.4.1 and then we say upgrade to 4.4.9. Lets just be clear, that in 4.4.X we recommend 4.4.9.

File: source/release_notes/5.0.3-2.rst [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      * If you are running any version of MongoDB 5.0.x or any version of MongoDB v4.4.2 or above, please follow closely all MongoDB Inc upstream recommendations regarding avoiding unclean shutdowns, check collections with the `validate <https://docs.mongodb.com/manual/reference/command/validate/>`_ command (watch for performance impacts), and make sure and resync any impacted nodes from unaffected nodes as recommended in `WT-7995 <https://jira.mongodb.org/browse/WT-7995>`_
```
Comment (spron-in) [inline source/release_notes/5.0.3-2.rst]: good one.

File: source/release_notes/5.0.3-2.rst [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+
+Release Highlights
+==================
+
+The bug fixes, provided by MongoDB and included in Percona Server for MongoDB, are the following:
+
+* `SERVER-58936 <https://jira.mongodb.org/browse/SERVER-58936>`_ - Unique Index constraint violations possible - can cause duplicate data - fixed in version 5.0.2.
```
Comment (spron-in) [inline source/release_notes/5.0.3-2.rst]: Shall we mention it here? It is 5.0.3 release notes, not 5.0.2.

File: source/release_notes/5.0.3-2.rst [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+* WT-7984 and associated Server Bug SERVER-60371.
+* `WT-7984 <https://jira.mongodb.org/browse/WT-7984>`_ - Bug that could cause Checkpoint thread to omit a page of data. If the server experiences an unclean shutdown, an inconsistent checkpoint is used for recovery and causes data corruption. Fixed in version 5.0.3.
+
+  Requires the `validate <https://docs.mongodb.com/manual/reference/command/validate/>`_  command to be run and possible data remediation via complete initial sync.
+* `SERVER-60371 <https://jira.mongodb.org/browse/SERVER-60371>`_ - Deployments, previously running version 4.4.8 and upgraded to 4.4.9, could still experience the Duplicate Key error and Fatal assertion. Related to the two previous Wired Tiger bugs.
```
Comment (spron-in) [inline source/release_notes/5.0.3-2.rst]: Any reason why we mention it here? How does it relate to 5.0.X ?

File: source/release_notes/5.0.2-1.rst [hunk 6]
```diff
@@ -11,6 +11,14 @@
 
    With a lot of new features and modifications introduced, we recommend using this release candidate in testing environments only.
 
+.. warning::
+
+   This version is not recommended for production use due to the following critical issues: `WT-7984 <https://jira.mongodb.org/browse/WT-7984>`_ and `WT-7995 <https://jira.mongodb.org/browse/WT-7995>`_. They are fixed in MongoDB 5.0.3 Community Edition and :ref:`5.0.3-2`.
```
Comment (igorsol) [inline source/release_notes/5.0.2-1.rst]: Not recommended to use because of issues fixed in this release? It seems something is wrong here.

Comment (igorsol) [reply source/release_notes/5.0.2-1.rst]: Probably rewording is necessary

**Review by igorsol — CHANGES_REQUESTED** [review]

File: source/release_notes/5.0.2-1.rst (hunk 6 above)
Comment (nastena1606) [reply source/release_notes/5.0.2-1.rst]: Explicitly added version to make it clearer which version is affected and in which issues are fixed

**Review by sandraromanchenko — APPROVED** [review]

**Review by spron-in — APPROVED** [review]

**Review by spron-in — APPROVED** [review]

File: source/release_notes/5.0.2-1.rst (hunk 6 above)
Comment (igorsol) [reply source/release_notes/5.0.2-1.rst]: Ah, sorry, now I see my mistake. Thank you for update

**Review by igorsol — APPROVED** [review]
LGTM

**Pull request merged**